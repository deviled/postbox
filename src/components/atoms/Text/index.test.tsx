import React, { FC } from 'react';
import { render } from '__tests__/utils';
import { Text } from '.';

describe('Text.tsx', () => {
  const Component: FC = () => (
    <Text color="white" bgcolor="error">
      Error input
    </Text>
  );

  it('should match snapshot', () => {
    const { baseElement } = render(<Component />);

    expect(baseElement).toMatchSnapshot();
  });
});
