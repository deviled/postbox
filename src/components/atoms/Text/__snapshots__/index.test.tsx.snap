// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text.tsx should match snapshot 1`] = `
.c0 {
  font-size: 1rem;
  font-weight: 400;
  color: #121212;
  color: #ffffff;
  background-color: #f44336;
}

<body>
  <div>
    <span
      class="c0"
      color="white"
    >
      Error input
    </span>
  </div>
</body>
`;
